### ISSUE TITLE
Ban an item next pixelmon update for wz

### ISSUE THREAD

**Issue by DragonBallBreakersRandomizer** [SmashMC-Development/Suggestions#939]
**Describe the suggestion**
so if you played pokemon Arceus, you may know of the new plate you get called the legend plate, if you don't here's a description I copied and pasted.
: The Legend Plate is a [Key Item] in [Pokémon Legends: Arceus]. It is one of the [Plates] which are associated with Arceus.
: When used on Arceus, the Legend Plate changes it to the Normal type, like the Blank Plate.

**In battle, Arceus gains the ability to change type when using Judgment, based on the selected opponent's type. While selecting a move, Judgment displays the type that it will become against the selected opponent if used that turn.**

Judgment will become a type that is super effective against the target, prioritizing any double weaknesses of dual-type targets.
If multiple types are equally effective, Judgment will change to a type that best resists the target's primary type, prioritizing immunities, regardless of what the target's secondary type may be.

My suggestion is to ban this item from wz completely and no one can bring one in their inventory if they tried to. If not then please make the method to obtain the plate harder like lower the odds as everyone's just gonna get a arceus and have one on their team, its completely unfair.
**Which servers does this apply to?**
ALL

**Additional context**
Maybe make the plate extra harder to get if you don't ban it.

**Comment by Mikirae** [comment]
Yeah we'll definitely need to ban that. That might be the dumbest thing I've ever seen Pokemon implement into any game lmfao
